part 2 audit of gov manual (#82)
radicle-foundation/radworks-governance: manual.md

Submitting a proposal on-chain is the final step in the governance process. An RGP can be submitted to on-chain governance by anyone who has an amount of $RAD greater than or equal to **1% of the total $RAD supply** delegated to their address (i.e. ≥ 1M $RAD). If a proposal author does not meet this requirement, they should reach out to the Governance Committee as soon as possible. 

For Submission, proposal authors are required to: 
- Post an updated version of their proposal on the forum that incorporates feedback from the _Discussion_ and _Formal Review_ phases. This version of the proposal should be posted under the [Active Proposals](https://community.radworks.org/c/governance/proposals/18) category on the forum.

> - Create an on-chain proposal using Tally or Boardroom.

Reviewer: Should we link to a "how to" guide? I'm sure Tally has one...

Author: Good idea - how about this one? https://docs.tally.xyz/knowledge-base/proposals/creating-proposals

> - Create an on-chain proposal using Tally or Boardroom.

Reviewer: ```suggestion
- [Create an on-chain proposal](https://docs.tally.xyz/knowledge-base/proposals/creating-proposals) using Tally or Boardroom. 
```

> - The forum post and on-chain poll should both be live no later than the fourth Tuesday of the proposal cycle (see [Proposaal Cycles](###ProposalTypes). 

Reviewer: ```suggestion
- The forum post and on-chain proposal should be submitted no later than the fourth Tuesday of the proposal cycle (see [Proposaal Cycles](###ProposalTypes). 
```

- Inlcude links to both the *Discsussion* and the *Formal Review* versions of the proposal in the final Submission post. Please use [this template]([url](https://github.com/radicle-foundation/radworks-governance/blob/main/templates/06-Submission-forum-post-header)) as a guide. 

> - Please double check that the executable on-chain code in correct! 

Reviewer: ```suggestion
- Please double check that the executable on-chain code is correct! 
```

### Proposal Cycles

> Radworks reviews proposals in **monthly cycles** to sync voting periods and establish a manageable cadence for governance participants. 

Reviewer: ```suggestion
Proposals are reviewed in **monthly cycles** to sync voting periods and establish a manageable cadence for governance participants. 
```

**Proposals can be posted to the forum for Discussion at anytime throughout the monthly cycle, but require at least 7 days on the forum to be considered for that month’s voting cycle. If proposal are posted after the second Monday deadline of a proposal cycle, they will be included in the next cycle. New proposals should be posted in the [Proposal Discussions](https://community.radworks.org/c/governance/discussions/19) sub-category under the _Governance_ category on the forum.**

### Proposal Review Calls

> Proposal Review are monthly calls take place on the second Wednesday of each month. These calls are a time for proposal authors to discuss their active proposals and answer questions from the community. The calls will be recorded and [archived](https://www.youtube.com/playlist?list=PLUUjDC9sOrpktWjO7jNFwsisK0vi5d_Tx) for reference. _Proposal authors are expected to make an effort to attend these calls and should come prepared to answer questions and discuss their proposal with attendees._

Reviewer: ```suggestion
Proposal Reviews are monthly calls take place on the second Wednesday of each month. These calls are a time for proposal authors to discuss their active proposals and answer questions from the community. The calls will be recorded and [archived](https://www.youtube.com/playlist?list=PLUUjDC9sOrpktWjO7jNFwsisK0vi5d_Tx) for reference. _Proposal authors are expected to make an effort to attend these calls and should come prepared to answer questions and discuss their proposal with attendees._
```

**Submission triggers a 3-day voting period. If passed, the proposal is queued for 48hrs, then it can be executed.**

## Delegation

> To participate in off-chain and on-chain voting, you have to first delegate the voting weight of your RAD. This can be done via various governance management platforms (e.g. [Tally](https://www.tally.xyz/gov/radworks) or [Boardroom](https://boardroom.io/radworks/proposals). After connecting your wallet to one of these platforms, you can choose to delegate the voting weight held in that wallet to the address (or addresses) of the token holder's choice:

Reviewer: ```suggestion
To participate in off-chain and on-chain voting, you must first delegate the voting weight of your $RAD. This can be done via various governance management platforms (e.g. [Tally](https://www.tally.xyz/gov/radworks) or [Boardroom](https://boardroom.io/radworks/proposals). After connecting your wallet to one of these platforms, you can choose to delegate the voting weight held in that wallet to the address (or addresses) of the token holder's choice:
```

- The owner’s own wallet, if they would like to vote on their own.
- Another user's wallet, if they would like the other user to vote on their behalf.

> Once delegation is completed, the voting weight of the RAD can be used to vote. See this [FAQ](https://docs.radworks.org/community/faq) for further delegation and voting trickss & tips. 

Reviewer: ```suggestion
Once delegation is completed, the voting weight of the RAD can be used to vote. See this [FAQ](https://docs.radworks.org/community/faq) for further information about delegation and voting.
```
